docs/environments/kind.md

```diff
@@ -0,0 +1,73 @@
+# `kind` - `k`ubernetes `in` `d`ocker
+
+[kind](https://github.com/kubernetes-sigs/kind) provides the quickest way to
+set up clusters for use with the Federation v2 control plane.
+
+## Download and Install kind
+
+If you don't yet have `kind` installed, you can run the following script to
+download and install a known working version. Make sure you have the `GOPATH` envi
```

Comment:
s/envi/env

Reply:
Removed as `GOPATH` is no longer required.